SignalMessaging/categories/UIColor+OWS.m

```diff
@@ -98,40 +98,6 @@ + (UIColor *)ows_messageBubbleLightGrayColor
     return [UIColor colorWithHue:240.0f / 360.0f saturation:0.02f brightness:0.92f alpha:1.0f];
 }
 
-+ (UIColor *)backgroundColorForContact:(NSString *)contactIdentifier
```

Comment:
pending myles approval, I think it's better to just get rid of these rather than to map legacy values to something similar.

Reply:
+1